drools-wb-screens/drools-wb-dsl-text-editor/drools-wb-dsl-text-editor-backend/src/main/java/org/drools/workbench/screens/dsltext/backend/server/indexing/DslFileIndexer.java

```diff
@@ -33,37 +31,29 @@
 import org.guvnor.common.services.project.model.Project;
 import org.kie.workbench.common.services.datamodel.backend.server.service.DataModelService;
 import org.kie.workbench.common.services.refactoring.backend.server.indexing.DefaultIndexBuilder;
-import org.kie.workbench.common.services.refactoring.backend.server.indexing.ErrorMessageUtilities;
-import org.kie.workbench.common.services.refactoring.backend.server.indexing.PackageDescrIndexVisitor;
-import org.kie.workbench.common.services.refactoring.backend.server.util.KObjectUtil;
+import org.kie.workbench.common.services.refactoring.backend.server.indexing.drools.AbstractDrlFileIndexer;
 import org.kie.workbench.common.services.refactoring.model.index.IndexElementsGenerator;
-import org.kie.workbench.common.services.refactoring.model.index.Rule;
-import org.kie.workbench.common.services.shared.project.KieProjectService;
+import org.kie.workbench.common.services.refactoring.model.index.Resource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.uberfire.backend.server.util.Paths;
-import org.uberfire.ext.metadata.engine.Indexer;
-import org.uberfire.ext.metadata.model.KObject;
-import org.uberfire.ext.metadata.model.KObjectKey;
-import org.uberfire.io.IOService;
 import org.uberfire.java.nio.file.Path;
 
+// TODO: yeah, so there's no "rule => <rule name>" key-value pair added to the lucene index doc here
```

Comment:
Yeah.. bit oh a hole in the new indexing :-(

Is the enumerated ```ResourceType``` sufficiently flexible to support all existing (and future) asset types?